documentation-open-api.yaml

```diff
@@ -230,7 +230,7 @@ definitions:
         description: 'Revenu net avant correction'
       montantImpot:
         type: integer
-        description: 'Montant des impots'
```

Comment:
To my opinion, we should put Mr Obled's mail here, with some adaptations perhaps.

Reply:
As seen IRL, we still have questions about how our API react to these to edge cases.